README.md

```diff
@@ -1,87 +1,81 @@
 
+## Deployment of the Ansible enviroment
 
-Ansible playbooks to deploy production instances on Irma from irma3. 
+Clone the repository to `/lupus/ngi/irma3/deploy`
 
-## Enabling the functionality of this repo
+Copy the file from the repository under `bootstrap/bashrc` to `/lupus/ngi/irma3/bashrc` 
 
-Checkout this repo to `/lupus/ngi/irma3/deploy`
+Run `source /lupus/ngi/irma3/bashrc` this file has to be sourced for virtually any operation relating to this repository.
 
-Copy the file `bootstrap/bashrc` to `/lupus/ngi/irma3/bashrc`. This file has to be sourced every time a user wants to work. 
+Setup a virtual environment, i.e: `/lupus/ngi/irma3/virtualenv-15.0.0/virtualenv.py -p /usr/bin/python2.7 /lupus/ngi/irma3/ansible-env`
 
-Setup a virtual enviroment, i.e: `/lupus/ngi/irma3/virtualenv-15.0.0/virtualenv.py -p /usr/bin/python2.7 /lupus/ngi/irma3/ansible-env`
+Activate the environment using source `/lupus/ngi/irma3/ansible-env/bin/activate`
 
-Activate the environment `source /lupus/ngi/irma3/ansible-env/bin/activate` 
-
-Install ansible `pip install ansible`
+Install ansible with `pip install ansible`
 
 Download Anaconda and install it to `/lupus/ngi/sw/anaconda`
 
-Enable rsync functionality `pip install pexpect`
+Manually set Anaconda's permissions with `chmod -R g+rwX /lupus/ngi/sw/anaconda/`
 
-## Developing new deployment scripts
-### Irma 3
-`source /lupus/ngi/irma3/bashrc` loads environment variables and sets GID to ngi-sw
+Enable rsync functionality by using `pip install pexpect`
 
-`source /lupus/ngi/irma3/ansible-env/bin/activate` loads python enviroment, for ansible and `sync.py`
+##Deployment of the NGI pipeline
 
-Write deployment scripts under `/lupus/ngi/irma3/deploy`. Make sure the target is somewhere under `/lupus/ngi/`.
-Some folders (such as `/lupus/ngi/irma3/`) should not be used as targets as they are never rsynced.
+###Requirements
+The following files need to be present on irma3 in order to successfully deploy a working version of NGI pipeline:
 
-Run the deployment script, for instance `ansible-playbook install.yml`
+- A working connection to charon or charon-dev as well as a generated API token. 
 
-Place any additional files that need to be synced over under `/lupus/ngi/`
+- A valid GATK key placed under `/lupus/ngi/irma3/deploy/files`. The filename must be specified in the gatk_keyvariable in `host_vars/127.0.0.1/main.yml`. 
 
-`python sync.py <remote dest>` rsyncs all files under `/lupus/ngi` on irma3 to irma1. Default directory is `/lupus/ngi`
+- A `charon_credentials.yml` file placed under `/lupus/ngi/irma3/host_vars/127.0.0.1/` listing the variables `charon_base_url`, `charon_api_token_upps` and `charon_api_token_sthlm`
 
+###Typical deployment
 
-## Running the supported tools
+Write deployment scripts under `/lupus/ngi/irma3/deploy`
```

Comment:
Should we mention that we write them under `devel` instead, and then fetch and merge in `deploy` when we're satisfied?